webrtc/RTCPeerConnection-getStats.https.html

```diff
@@ -40,14 +40,12 @@
         2.  Resolve p with the resulting RTCStatsReport object, containing the gathered stats.
    */
   promise_test(t => {
-    const pc = new RTCPeerConnection();
-    t.add_cleanup(() => pc.close());
+    const pc = createPeerConnectionWithCleanup(t);
```

Comment:
There was a pass done earlier that removed use of this helper, presumably in the service of making tests more obvious to people that didn't know the helpers by heart.
Consider reverting this part of the change (or rebasing).
(I think @jan-ivar did that change).